src/pages/Governance/VotingWallet/testIds.ts

```diff
@@ -1,6 +1,8 @@
 export default {
   votingWeightValue: 'vote-voting-wallet-voting-weight-value',
   totalLockedValue: 'vote-voting-wallet-total-locked-value',
+  delegateButton: 'delegate-button',
   delegateYourVoting: 'vote-voting-wallet-delegate-your-voting',
   depositYourTokens: 'vote-voting-wallet-deposit-your-tokens',
+  votingDisabledWarning: 'voting-disabled-warning',
```

Comment:
I recommend adding the same prefixes as other test IDs as they need to be unique within the entire app:
```suggestion
  delegateButton: 'vote-voting-wallet-delegate-button',
  delegateYourVoting: 'vote-voting-wallet-delegate-your-voting',
  depositYourTokens: 'vote-voting-wallet-deposit-your-tokens',
  votingDisabledWarning: 'vote-voting-wallet-voting-disabled-warning',
```